types/vanilla-masker/index.d.ts

```diff
@@ -3,6 +3,10 @@
 // Definitions by: BenLorantfy <https://github.com/BenLorantfy>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+interface Element { }
+interface Node { }
+interface NodeListOf<TNode = Node> { }
```

Comment:
I'm not sure why this is needed?  `Element`, `Node`, and `NodeListOf` should be built into the typescript standard lib